### DISCUSSION TITLE
for IEA15MW_03

### DISCUSSION THREAD

**Discussion by liangzhi85** [IEAWindTask52/LidarAssistedControl#19] [Q&A]
Dear IEA LAC,
I use the script of the RunExample_4BeamPulsed.m in "Release\IEA15MW_03", there is a variation for the VLOS01LI, please find it by below figure. May I ask why this happens?
thanks in advance! have a great day!
![1719990809441](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/34777b77-0f42-452a-bcfa-4f2a0138782f)

**Comment by liangzhi85** [comment]
I try to see VLOS at other beams, please find by the below figure.
thanks in advance for your help!
![image](https://github.com/IEAWindTask52/LidarAssistedControl/assets/123851926/f5e7bf72-6f07-44b3-9d5a-1ccf94a29bfd)

**Comment by DavidSchlipf** [comment]
Dear Zhi Liang,
sorry for the delay. The signal VLOS01LI is the line-of-sight wind speed from the first distance. Since the BeamID is changing, also the line-of-sight is changing. The other distances are available via the Bladed interface, see Fortran code LDP_v1_Subs.f90 line 154:
LidarVar%v_los(LidarVar%IndexGate) 
We updated the simulation files on github and now all the other distances are also available in the output files in the Channels VLOSxxLI with xx from 01 to 10.
Hope this helps.
David
